Minor updates to access review beta to expose top-level decisions navigation route (#15036)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md

---
title: "accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource resource type"
description: "Every decision item in an access review represents a principal's access to a resource. accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource represents that the decision item represents an access package assignment policy."
author: "isabelleatmsft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

> Every decision item in an access review represents a principal's access to a resource. All accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource objects represent access to access package assignment policies. accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource is an open type that allows for other properties to be passed in.

Reviewer: @shubhamguptacal did i capture this right?

Author: This makes sense.

> Every decision item in an access review represents a principal's access to a resource. All accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource objects represent access to access package assignment policies. accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource is an open type that allows for other properties to be passed in.

Reviewer 2: ```suggestion
Represents a principal's access to an access package assignment policy. accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource is an open type that allows other properties to be passed in.

Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).
```

## Properties
|Property|Type|Description|
|:---|:---|:---|

> |displayName|String|Display name of the resource|
> |id|String|Resource ID|
> |type|String|Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.|

Reviewer 2: ```suggestion
| displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
| id | String | Identifier of the decision item resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
| type | String | Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```

> |type|String|Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.|

Reviewer: @shubhamguptacal shouldnt we only include the access package one as the type option here? and likewise for all the other files

Author: Yes, that makes sense.

> |accessPackageId|String| Access Package Id |
> |accessPackageDisplayName|String| Access Package Display name |

Reviewer 2: ```suggestion
|accessPackageId|String| Identifier of the access package to which access has been granted. |
|accessPackageDisplayName|String| Display name of the to which access has been granted. |
```

## Relationships
None.

## JSON representation
The following is a JSON representation of the resource.
<!-- {

>   "blockType": "resource",
>   "@odata.type": "microsoft.graph.accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource",
>   "openType": true

Reviewer 2: ```suggestion
  "blockType": "resource",
  "@odata.type": "microsoft.graph.accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource",
  "baseType": "microsoft.graph.accessReviewInstanceDecisionItemResource",
  "openType": true
```

---
title: "accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource resource type"

> description: "Represents a principal's access to an access package assignment policy. "

Reviewer: @FaithOmbongi : I think the wording of the remaining suggestions is slightly inaccurate where it says access has been **granted** through an ARIDI object. I would say access is **represented** through that object. 
How do you feel about: "Represents an access package assignment policy for which access is represented through an accessReviewInstanceDecisionItem object."

> description: "Represents a principal's access to an access package assignment policy. "

Reviewer 2: Your change sounds good, @isabelleatmsft. Go for it.

author: "isabelleatmsft"
ms.localizationpriority: medium
ms.prod: "governance"
doc_type: resourcePageType
---

# accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

> Represents a principal's access to an access package assignment policy. accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource is an open type that allows other properties to be passed in.

Reviewer 2: ```suggestion
Represents an access package assignment policy to which access has been granted through an [accessReviewInstanceDecisionItem]((https://docs.microsoft.com/en-us/graph/api/resources/accessreviewinstancedecisionitem?view=graph-rest-beta) object. **accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource** is an open type that allows other properties to be passed in.
```

Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).

## Properties
|Property|Type|Description|
|:---|:---|:---|

> | displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|

Reviewer: ```suggestion
| displayName | String | Display name of the access package. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
```

| id | String | Identifier of the decision item resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |

> | type | String | Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |

Reviewer: ```suggestion
| type | String | Type of resource. Type will always be `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```
